mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -138,6 +138,7 @@ final class EMLDAOptimizer extends LDAOptimizer {
     this.docConcentration = if (docConcentration == -1) (50.0 / k) + 1.0 else docConcentration
     this.topicConcentration = if (topicConcentration == -1) 1.1 else topicConcentration
     val randomSeed = lda.getSeed
+    docs.persist()
```

Comment:
`docs` is user input hence persist not required.
https://github.com/apache/spark/blob/80fbc382a60973db21367a922a0fb797e5ab382d/mllib/src/main/scala/org/apache/spark/mllib/clustering/LDA.scala#L331